### DISCUSSION TITLE
Kineticist rod bug fix

### DISCUSSION THREAD

**Discussion by Checkflank** [Truinto/DarkCodex#271] [Ideas]
So I noticed that activating the kineticist rod bugs some abilities out in game. Is there any possibility that we could see a possible fix for that?

**Comment by Truinto** [comment]
Not with that kind of description. Which abilities? Which rods? What is the expected and actual behavior?
In general, rods apply metamagic via the event manager. Stuff like empower or maximize. There is very little that could go wrong with that.
